CMakeLists.txt

```diff
@@ -4,8 +4,7 @@
 
 cmake_minimum_required(VERSION 2.8)
 
-SET(PROJECTNAME wholeBodyModel)
-PROJECT(${PROJECTNAME})
+PROJECT(wholeBodyModel)
```

Comment:
do we want to call the project wholeBodyModel and not mexWholeBodyModel? Wont this be confusing?